### ISSUE TITLE
Improvements

### ISSUE THREAD

**Issue by alberto-bottarini** [emilianotisato/nova-tinymce#5]
Hi, I think these 2 improvements should be useful:

- hide field by default in grid view (as textarea fileld are)
- use the "Show content" trigger, to show field value in detail view

Thanks

**Comment by emilianotisato** [comment]
@alberto-bottarini you are right! Can you work on a PR and I merge it?? I am with zero time at this days...

**Comment by alberto-bottarini** [comment]
https://github.com/emilianotisato/nova-tinymce/pull/6 and https://github.com/emilianotisato/nova-tinymce/pull/7
